### DISCUSSION TITLE
Download .mpga file not .mp3

### DISCUSSION THREAD

**Discussion by mahdiar** [vidstack/player#1255] [Q&A]
Hi
I add download link to my audio player. When I click on the button it save the file as .mpga not .mp3.
Is there anyway to change it?
Regards

**Comment by mihar-22** [comment]
You can provide an alternative file name. I don't know what framework or layout you're using so please provide more information in the future.

If React, like so:

```tsx
<DefaultVideoLayout download={{ url: '...', filename: 'file.mpga',  }}>
```

**Reply by mahdiar** [reply]
Thanks I use in HTML
